### DISCUSSION TITLE
Welcome to tailbliss Discussions!

### DISCUSSION THREAD

**Discussion by nusserstudios** [nusserstudios/tailbliss#57] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dombyte** [comment]
Hey im Dominik and I want to use this theme because i think it looks amazing. Now to my kida problem, how can I implement this theme that i'm able to pull changes from the tailbliss main branch but also have my own branches including the main one where I can develope. I was asking because this is not implemntet like a normal Hugo Theme in the themes directory which i can add as an submodule. I hope someone could assist me with that.

**Reply by nusserstudios** [reply]
Thanks Dominik, I've had several people ask about this. This was my first go with Hugo, I plan to get it setup like a real theme, when I have some time. It's definitely a Hugo site, and not a theme and it's front-end dev heavy right now. I really just wanted to share some of my Tailwind Blocks to make it easier for copy/paste. I am pretty bogged down with real world client work at the moment, (I'm a full-time free lancer). I don't want to give you a timeframe, but I will do my best to get it ported to a theme with exampleSite dir as soon as I can.

**Reply by dombyte** [reply]
Sounds great! Take your time.
